extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -329,8 +330,9 @@ export async function getCompletionItemsFromSpecs(specs: Fig.Spec[], terminalCon
 
 	if (!specificSuggestionsProvided && (filesRequested === foldersRequested)) {
```

Comment:
I think `specificSuggestionsProvided` is badly named here? Does it actually mean an argument was provided? We want availableCommands to get filled whenever the command is being input, I think that condition would be:

```ts
commandLine.substring(0, cursorIndex).trimStart().includes(' ')
```